doc/Learn/Mvux/WorkingWithRecords.md

```diff
@@ -106,7 +106,7 @@ In our Model:
 message = message with
 {
     Content = response.Message,
-    Status = response.IsError ? Status.Error : Status.Value
+    status = response.IsError ? Status.Error : Status.Value
```

Comment:
I think it would be best instead to alter the property declarations here: https://github.com/JohannesMP/uno.extensions/blob/5e2e4a67c36c9cfef6fcbf21a2d393166334ed35/doc/Learn/Mvux/WorkingWithRecords.md?plain=1#L98

And change it to properly capitalize each of those public properties on the `record`. So it should be 

```csharp
public partial record Message(string Content, Status Status, Source Source);
```